Neos.Flow/Classes/Http/Client/InternalRequestEngine.php

```diff
@@ -115,6 +135,10 @@ public function sendRequest(ServerRequestInterface $httpRequest): ResponseInterf
         $this->validatorResolver->reset();
 
         $objectManager = $this->bootstrap->getObjectManager();
+
+        /**
+         * @var Http\Middleware\MiddlewaresChain $middlewaresChain
+         */
```

Comment:
Funny… I don't need this in PhpStorm, the type of `$middlewaresChain` is still known… 🤔